packages/data/single-file/src/es6-promise.ts

```diff
@@ -0,0 +1 @@
+export default Promise;
```

Comment:
Missing `.auto()` and `/auto`: https://www.npmjs.com/package/es6-promise?activeTab=readme#auto-polyfill

Reply:
`.polyfill()` is added, but I don't know how to add `/auto` here, seems need another file?

Reply:
use cjs style is because <https://stackoverflow.com/questions/68833139/cannot-export-buffer-only-local-declarations-can-be-exported-from-a-module>, I don't want to modify tsconfig.